build-support/bin/generate_docs.py

```diff
@@ -227,6 +236,13 @@ def _get_id(self, url) -> str:
         """Returns the id of the entity at the specified readme.io API url."""
         return cast(str, self._access_readme_api(url, "GET", "")["_id"])
 
+    @classmethod
+    def _render_parent_page_body(cls, items: Iterable[str], sync: bool) -> str:
+        """Returns the body of a parent page for the given items."""
+        # The page just lists the items, with links to the page for each one.
+        lines = [f"**[{item}]({cls._link(item, sync)})**\n" for item in items]
```

Comment:
This style contrasts with how we've done index pages elsewhere, like this: https://www.pantsbuild.org/v2.0/docs/target-api.

Personally, I prefer the prior convention of a bulleted list without bolding. I find bolding distracting if you don't have other content to differentiate it. The new lines make it harder to fit on a screen.